Assets/__Scripts/BeatSaberSong.cs

```diff
@@ -407,7 +407,8 @@ public static BeatSaberSong GetSongFromFolder(string directory)
                                 beatmap.UpdateName(d["_beatmapFilename"]);
                                 set.difficultyBeatmaps.Add(beatmap);
                             }
-                            set.difficultyBeatmaps = set.difficultyBeatmaps.OrderBy(x => x.difficultyRank).ToList();
+                            // If there are already difficulties ignore duplicates of the same difficulty
+                            set.difficultyBeatmaps = set.difficultyBeatmaps.GroupBy(it => it.difficultyRank).Select(group => group.First()).OrderBy(x => x.difficultyRank).ToList();
```

Comment:
I do have a [DistinctBy](https://github.com/Caeden117/ChroMapper/blob/dev/Assets/__Scripts/EnumerableExtensions.cs#L6) extension method, could this be used instead of `.GroupBy().Select()`?

Reply:
Also updated